src/ImgixTag.js

```diff
@@ -49,10 +49,14 @@ var ImgixTag = (function() {
   }
 
   ImgixTag.prototype._extractBaseParams = function() {
-    var params;
+		var params = {};
+
+		if (this.settings.defaultParams) {
```

Comment:
Should we type-check this to make sure it's an object?

Reply:
Yep, I can do this